rkt/app_add.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/coreos/rkt/common"
+	"github.com/coreos/rkt/common/apps"
+	pkgPod "github.com/coreos/rkt/pkg/pod"
+	"github.com/coreos/rkt/rkt/image"
+	"github.com/coreos/rkt/stage0"
+	"github.com/coreos/rkt/store/imagestore"
+	"github.com/coreos/rkt/store/treestore"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdAppAdd = &cobra.Command{
+		Use:   "add UUID IMAGEID ...",
+		Short: "Add an app to a pod",
+		Long:  `This allows addin an app that's present on the store to a running pod`,
+		Run:   runWrapper(runAppAdd),
+	}
+)
+
+func init() {
+	cmdApp.AddCommand(cmdAppAdd)
```

Comment:
Wishlist: this seems to be missing a `--uuid-file` for specifying the target pod.